.gitignore

```diff
@@ -18,3 +18,7 @@ node_modules/
 package.json
 .ipynb_checkpoints/
 .jekyll-cache/
+# pixi environments
+.pixi
+*.egg-info
+!pixi.lock
```

Comment:
With `pixi` projects you want the lock file under version control. At the top of the existing file lock files of all type are ignored

https://github.com/iris-hep/iris-hep.github.io/blob/cfcc0405fb78adee464189a25e19c8dd10c56a9e/.gitignore#L1

but [`Gemfile.lock` is under version control](https://github.com/iris-hep/iris-hep.github.io/blob/cfcc0405fb78adee464189a25e19c8dd10c56a9e/Gemfile.lock) so maybe @henryiii added them to `.gitignore` so that people wouldn't be accidentally committing changes, forcing people to be clear on what they were doing?

Reply:
IIRC that was the reasoning.

Reply:
Cool. Unless you're strongly against it, I think I might keep in the `!pixi.lock` as this should be static unless people add a dependency or `rm pixi.lock` and resolve the environment. I could go either way though.

Reply:
No, it's fine, and in fact we could just swap this for Gemfile.lock instead of the all-lock-files exclusion.

Reply:
Good idea. I split that out in the rebase I just did.